parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -183,10 +177,13 @@ void forwardConfig(String parquetProperty, String icebergProperty, String defaul
           conf.set(entry.getKey(), entry.getValue());
         }
 
-        long rowGroupSize = Long.parseLong(config.getOrDefault(
-            PARQUET_ROW_GROUP_SIZE_BYTES, PARQUET_ROW_GROUP_SIZE_BYTES_DEFAULT));
+        ParquetProperties parquetProperties = ParquetProperties.builder()
```

Comment:
Looks like this dropped `withWriterVersion` that was set in `ParquetWriter`.

Reply:
I let this default as the `else` path defaults as well (so both use V1 by default). Thinking I'll make this more explicit and just have a constant in this class hardcoded to V1 then for now and both paths will use that.

Reply:
Not sure if this should be configurable though? Will hardcode it to V1 for now.

Reply:
Looks sort of weird just hardcoding it but left it like this for future logic that wants to detect it in some way e.g. explicitly set or via iceberg properties